server/mailet/mock-smtp-server/src/test/java/org/apache/james/mock/smtp/server/MockSMTPServerTest.java

```diff
@@ -23,6 +23,7 @@
 import static org.apache.james.mock.smtp.server.Fixture.BOB;
```

Comment:
I'm missing a test with a "condition":

When "bob" 421 without reject, and alice 250, then a mail is stored with only alice in the rcpt field.